src/components/Explore/NowPlaying.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from "@emotion/css";
+import React from "react";
+import { fetchLatestTracks } from "../../services/Api";
+import usePagination from "../../utils/usePagination";
+import { CenteredSpinner } from "../common/Spinner";
+import TrackList from "../common/TrackList";
+
+export const NowPlaying: React.FC = () => {
+  const [tracks, setTracks] = React.useState<Track[]>([]);
+
+  const { results, isLoading } = usePagination<Track>({
+    apiCall: React.useCallback(fetchLatestTracks, []),
+    options: { limit: 2, order: "plays" },
```

Comment:
```suggestion
    options: React.useMemo(() => ({ limit: 2, order: "plays" }), []),
```

React has a hard time detecting if objects are the same, and so on every render this would be passing a new object and React wouldn't know, causing a change in `results` causing an infinite loop.